lib/assets/javascripts/turbolinks.js.coffee

```diff
@@ -140,7 +141,7 @@ changePage = (doc, options) ->
     nodesToChange = [currentBody]
 
   triggerEvent EVENTS.BEFORE_UNLOAD, nodesToChange
-  document.title = title
+  document.title = title if title != false
```

Comment:
> I'd prefer CoffeeScript's equality sugar here. `document.title = title if title isnt false` or `document.title = title unless title is false`

AFAIK we don't have a convention to use one syntax or the other in Turbolinks (I prefer `!=` but don't feel strongly about it). There's a mix of both in that file. We can settle on one when I add a `CONTRIBUTING.md` if you want.